.github/workflows/find-inactive-collaborators.yml

```diff
@@ -0,0 +1,17 @@
+name: Find inactive collaborators
+
+on:
+  schedule:
+    # Run on the 15th day of the month at 4:05 AM.
+    - cron: '5 4 15 * *'
```

Comment:
Out of curiosity, is the time of the day arbitrary? Also, since the comment is explaining the cron syntax, it might be worth adding that this is UTC (unlike most cron implementations).

Reply:
The only reason for choosing a somewhat oddball time like 4:05 AM is that my distant past as a systems administrator taught me to resist putting all the recurring cron jobs at midnight so they don't all run at once, taxing the system. But that was before cloud, containers, etc., and I'm not sure if that matters much or at all in this case. But old habits die hard.

Adding that it's UTC makes sense to me. I'll do that.